### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following:

### DISCUSSION THREAD

**Discussion by ps2109** [community/community#53501] [GitHub Education]
I am writing to you as a student who has applied for the Student Developer Pack on your platform, but unfortunately, my application has not been approved yet. I have followed all the instructions and provided all the required information, but I am still getting back to same screen  

**Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.**

can anyone look and help me to get it approved and uploading all the valid documents and also updated the profile as suggested but still same error again and again

**Comment by samus-aran** [comment]
Hi @ps2109 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. 

Hope this helps! Let us know how you get along 👍

**Reply by Saif221844** [reply]
There is no camera option, plus , what about desktop users ??

**Reply by samus-aran** [reply]
@Saif221844 There is camera options :) This thread is out of date now I would suggest reviewing more recently created discussions.

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for.
